fast_api_als/database/db_helper.py

```diff
@@ -16,6 +16,19 @@
     write a commong function that logs this response code with appropriate context data
 """
 
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+formatter = logging.Formatter('%(asctime)s %(levelname)s-%(message)s')
+
+file_handler = logging.FileHandler('logs.log')
+file_handler.setFormatter(formatter)
+
+logger.addHandler(file_handler)
+
+def log_db_self_table(res, operation):
+    response = res['ResponseMetadata']['HTTPStatusCode']
+    logging.info(f'Operation performed is {operation} with response code {response}')
```

Comment:
Good

Reply:
where you're using this function tho?
also add some logging in DBHelper's function.